sdk/spring/azure-spring-boot/src/test/java/com/azure/spring/aad/webapp/WebApplicationContextRunnerUtils.java

```diff
@@ -12,25 +12,29 @@
 
 import java.util.Optional;
 
-public class PropertiesUtils {
+public class WebApplicationContextRunnerUtils {
 
-    public static WebApplicationContextRunner getContextRunner() {
+    public static WebApplicationContextRunner getContextRunnerWithRequiredProperties() {
```

Comment:
Suggestion:
1. Rename `getContextRunnerWithoutRequiredProperties` to `getContextRunner`
2. Call `getContextRunner` in `getContextRunnerWithRequiredProperties`